runelite-client/src/main/java/net/runelite/client/ui/PluginPanel.java

```diff
@@ -70,17 +69,20 @@ public PluginPanel()
 		wrappedPanel.add(scrollPane, BorderLayout.CENTER);
 	}
 
+	public PluginPanel(boolean disableScrolling) {
```

Comment:
You can just do this directly by calling getScrollPane().setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_NEVER);
instead of creating multiple constructors. Or even by adding protected method. New constructor is probably bad idea.